modules/plugins/assistant/avante/avante-nvim.nix

```diff
@@ -5,7 +5,7 @@
 in {
   options.vim.assistant = {
     avante-nvim = {
-      enable = mkEnableOption "complementary Neovim plugin for avante.nvim";
+      enable = mkEnableOption "Complementary Neovim plugin for avante.nvim";
```

Comment:
The issue with this is that `mkEnableOption` prepends `Whether to enable` at the beginning of the string passed to it, so the result in documentation becomes

```
Whether to enable Complementary ...
```

This also applies to other options.